src/Views/Format.elm

```diff
@@ -63,6 +64,48 @@ formatFoodSelectedImpactPerKg { trigram, unit, decimals } totalMass =
         >> formatRichFloat decimals (unit ++ "/kg")
 
 
+formatFoodSelectedImpactScore : Impact.Definition -> Mass -> Impacts -> Html msg
+formatFoodSelectedImpactScore { trigram } totalMass impacts =
```

Comment:
Let's suppose for a minute we want to expose this score to the API; would we be happy with an HTML only function? Even if that's not a requirement atm, wouldn't it be nice to have the computation extracted to a more agnostic function retyurning an `Int`, with maybe a few unit tests added?